examples/echo_server/migrations/3_chainlink_oracle.js

```diff
@@ -0,0 +1,5 @@
+let Oracle = artifacts.require("../node_modules/smartcontractkit/chainlink/solidity/contracts/Oracle.sol");
```

Comment:
Now that we're in one repo, we can require the original at `artifacts.require("../../../solidity/contracts/Oracle.sol");` and not rely on `node_modules`. We shouldn't require from `node_modules/` for any of our migrations or contracts.